### PR TITLE
[oneDNN] Added Unit Test for Multiple instances prediction

### PR DESCRIPTION
**Pull request by jczaja**
### PR types
Others 

### PR changes
Others

### Describe
This is a unit tests testing an issue described #28459.

### CODE REVIEW

**Comment by jczaja** [comment]
@arlesniak , @jakpiase  Please review

**Review by jakpiase — DISMISSED** [review]
C++ part looks good. I don't have enough knowledge to speak about cmake code.

**Review by lidanqing-vv — DISMISSED** [review]
LGTM

**Comment by jczaja** [comment]
@luotao1 Could you please start your review? CI-APPROVAL will not pass without your colleagues approval

**Review by luotao1 — DISMISSED** [review]

File: paddle/fluid/inference/tests/api/CMakeLists.txt [hunk 1]
```diff
@@ -636,6 +647,7 @@ set_tests_properties(test_analyzer_googlenet PROPERTIES TIMEOUT 120)
 set_tests_properties(test_analyzer_small_dam PROPERTIES TIMEOUT 120)
 set_tests_properties(test_analyzer_transformer PROPERTIES TIMEOUT 120)
 set_tests_properties(test_analyzer_bert PROPERTIES TIMEOUT 120)
+set_tests_properties(test_analyzer_multi_model_prediction PROPERTIES TIMEOUT 120)
```
Comment (luotao1) [inline paddle/fluid/inference/tests/api/CMakeLists.txt]:
```
2020-12-08 23:59:52 0. You must have one QA (XieYunshen(Recommend) or chalsliu) approval for setting parameter RUN_TYPE as EXCLUSIVE, DIST, NIGHTLY, EXCLUSIVE:NIGHTLY or DISTNIGHTLY, or setting TIMEOUT properties.
2020-12-08 23:59:52 The corresponding lines are as follows:
2020-12-08 23:59:52 paddle/fluid/inference/tests/api/CMakeLists.txt
2020-12-08 23:59:52 +set_tests_properties(test_analyzer_multi_model_prediction PROPERTIES TIMEOUT 120)
2020-12-08 23:59:52 For more information, please refer to:https://github.com/PaddlePaddle/Paddle/wiki/PaddlePaddle-Unit-test-specification
2020-12-08 23:59:52 There are 1 approved errors.
```
I see this unit-test run very fast
```
2020-12-09 00:57:02 115/215 Test  #232: test_analyzer_multi_model_prediction .............   Passed    7.73 sec
```
You can remove line 650

Comment (jczaja) [reply paddle/fluid/inference/tests/api/CMakeLists.txt]: done

**Comment by jczaja** [comment]
@luotao1 Ok. I have removed timout for this test as asked. Please continue with your review

**Review by luotao1 — APPROVED** [review]

**Pull request merged**